service/raspbian-install-device-agent.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
+
+if [[ $EUID -ne 0 ]]; then
+    echo "This script must be run as root"
+    exit 1
+fi
+
+# Node.js version
+MIN_NODEJS=16
```

Comment:
We technically support NodeJS 14 for the device agent

Reply:
```suggestion
MIN_NODEJS=14
```